### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by shizotoaster**

### CODE REVIEW

File: src/main/resources/assets/emi/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+  "config.emi.ui.append_item_mod_id": "Добавлять ID Предмета из Мода",
+  "config.emi.ui.empty_search_craftable": "Показывать Создаваемые при Пустом Поиске",
+  "config.emi.ui.vertical_margin": "Ширина поля",
+  "config.emi.ui.miscraft_prevention": "Предотвращение Случайных Крафтов",
+  "config.emi.ui.fluid_unit": "Блоки Жидкостей",
+  "config.emi.ui.show_cost_per_batch": "Показывать Цену за Несколько",
+  "config.emi.ui.recipe_default_button": "Стандартная Клавиша Рецептов",
+  "config.emi.ui.recipe_tree_button": "Клавиша Дерева Рецептов",
+  "config.emi.ui.recipe_fill_button": "Клавиша Заполнения Рецептов", !!
+  "config.emi.ui.use_batched_renderer": "Использовать Batched Отрисовку", !!
```
Comment (unascribed) [inline src/main/resources/assets/emi/lang/ru_ru.json:47]: Syntax errors (`!!` is not valid JSON)

File: src/main/resources/assets/emi/lang/ru_ru.json [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+  "tag.c.glass_panes.orange": "Оранжевые Стеклянные Панели",
+  "tag.c.glass_panes.magenta": "Пурпурные Стеклянные Панели",
+  "tag.c.glass_panes.light_blue": "Голубые Стеклянные Панели",
+  "tag.c.glass_panes.yellow": "Желтые Стеклянные Панели",
+  "tag.c.glass_panes.lime": "Лаймовые Стеклянные Панели",
+  "tag.c.glass_panes.pink": "Розовые Стеклянные Панели",
+  "tag.c.glass_panes.gray": "Серые Стеклянные Панели",
+  "tag.c.glass_panes.light_gray": "Светло-серые Стеклянные Панели",
+  "tag.c.glass_panes.cyan": "Бирюзовые Стеклянные Панели",
+  "tag.c.glass_panes.purple": "Фиолетовые Стеклянные Панели,
```
Comment (unascribed) [inline src/main/resources/assets/emi/lang/ru_ru.json:340]: Syntax error (no closing ")

**Review by unascribed — CHANGES_REQUESTED** [review]
Once-over with a machine translator looks OK (due diligence, checking for profanity/etc) - a couple syntax errors though

File: src/main/resources/assets/emi/lang/ru_ru.json (hunk 1 above)
Comment (shizotoaster) [reply src/main/resources/assets/emi/lang/ru_ru.json:47]: Yeah, sorry, forgot to remove marks of things I wanted to translate later

**Review by unascribed — APPROVED** [review]

**Pull request merged**